### DISCUSSION TITLE
Styling <Select> chevron

### DISCUSSION THREAD

**Discussion by spidertwin2** [chakra-ui/chakra-ui#3104] [Q&A] [answered]
Hii,

I want the default color of `<Select>` chevron icon to be `gray.400`.  But, custom styling in theme.js is not working.

theme.js

```
const Select = {
  parts: ["icon"],
  baseStyle: {
    icon: {
      color: "gray.400",
    },
  },
  defaultProps: {
    size: "sm",
  },
};

const customTheme = extendTheme({
  components: {
    Select,
  },
});
```

Thank you :smile:

**Comment by dodas** [comment, marked as answer]
Hello @spidertwin2,

this has been fixed in #3129 and the exact code you posted works since v1.1.6.

Cheers!

**Reply by spidertwin2** [reply]
Just tried it, works! Many thanks :pray:
